src/components/LayoutHeader/Nav/LinkItems/index.tsx

```diff
@@ -102,7 +102,13 @@ const LinkItems: React.FC = () => {
       {navLinkItemsData.map((item, i) => {
         const popup = isPopup(item) ? popups[item.popupName] : undefined
         return (
-          <li key={i} className={styles.linkItem} ref={popup?.containerEl}>
+          <li
+            key={i}
+            className={styles.linkItem}
+            ref={popup?.containerEl}
+            onMouseEnter={popup?.open}
```

Comment:
Have either Ivan or Yaroslav agreed on changing the dropdowns to activate on hover? Not a blocker since this is a minor change and we can always change it back later :)